Source/Core/Core/HW/DSPHLE/UCodes/AX.h

```diff
@@ -62,7 +62,7 @@ enum AXMixControl
   // clang-format on
 };
 
-class AXUCode : public UCodeInterface
+class AXUCode /* not final */ : public UCodeInterface
```

Comment:
I think it might be better to have a comment above the class stating why it's not final, rather than a comment. Normally an unmarked class is nothing major, but I think indicating a class as non final through a comment should warrant a little elaboration to be explicit

Reply:
How does this look?

```C++
class AXUCode /* not final: subclassed by AXWiiUCode */ : public UCodeInterface
```

Reply:
> How does this look?
> 
> ```c++
> class AXUCode /* not final: subclassed by AXWiiUCode */ : public UCodeInterface
> ```

I think @lioncash means 
```c++
//This class is not final, due to [reason]
class AXUCode : public UCodeInterface
```